packages/salesforcedx-vscode-core/src/index.ts

```diff
@@ -497,8 +497,9 @@ export async function activate(context: vscode.ExtensionContext) {
   orgList.displayDefaultUsername(defaultUsernameorAlias);
   context.subscriptions.push(registerOrgPickerCommands(orgList));
 
-  // await setupOrgBrowser(context, defaultUsernameorAlias);
-  vscode.commands.executeCommand('setContext', 'sfdx:display_tree_view', false);
+  if (sfdxCoreSettings.showOrgBrowser()) {
```

Comment:
I think we should remove checking for the setting here. Having the flag being used in the `when` clause for the tree view will immediately show the option once you modify the setting file but the tree view itself will throw an error if you try to use it since the retrieve commands are not registered since `setupOrgBrowser` didn't run in activation.

Reply:
Sounds good, we can remove the check.